### PR TITLE
Mark TabError as noqa to silence linters on Python 3

### PR DESCRIPTION
**Pull request by cclauss**
flake8 testing of https://github.com/satwikkansal/wtfpython on Python 3.6.2

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./mixed_tabs_and_spaces.py:5:18: E999 TabError: inconsistent use of tabs and spaces in indentation
	return sum_so_far
                 ^
```

### CODE REVIEW

**Comment by satwikkansal** [comment]
Hey @cclauss Thanks for the PR. I'm wondering if there is some way to skip this statement for other popular linters like pylint.

**Comment by cclauss** [comment]
_<my code goes here>_  # noqa # pylint: disable={whatever}
* The first one tells pyflakes and flake8 to ignore
* The second one tells pylint to ignore -- however, I do not think pylint sees this kind of thing.

File: mixed_tabs_and_spaces.py [hunk 1]
```diff
@@ -2,6 +2,6 @@ def square(x):
     sum_so_far = 0
     for counter in range(x):
         sum_so_far = sum_so_far + x
-	return sum_so_far  # noqa Python 3 will raise a TabError here
+	return sum_so_far  # noqa: E999 # pylint: disable=bad-indentation Python 3 will raise a TabError here
```
Comment (satwikkansal) [inline mixed_tabs_and_spaces.py]: `# pylint: disable=bad-indentation` doesn't suppress the warning, instead, `# pylint: disable=mixed-indentation` works for me

Comment (satwikkansal) [reply mixed_tabs_and_spaces.py]: Also could you please replace `counter` with `_` as it's an unused variable and pylint is raising a warning about it :sweat_smile:

**Comment by satwikkansal** [comment]
Thanks @cclauss, merging it!

**Pull request merged**